src/ui/src/partials/system_view.html

```diff
@@ -16,9 +16,17 @@ <h1 class="page-header">
     <table
         datatable="ng"
         dt-options="dtOptions"
+        dt-instance="dtInstance"
         valign="center"
-        class="table table-striped table-bordered w-100">
+        class="table table-striped table-bordered w-100"
+        id="tableS">
       <thead>
+        <label style="top: 183px;left: 1540px; font-size: 14px; font-weight: normal; display: block; position: absolute; z-index:4;" >
```

Comment:
For things that have a bunch of styles like this it's easier to define a class and put them in the custom.css file. That'll also be cleaner since you can just specify the class here and on the other page that uses this.